portal/patient/templates/OnsiteDocumentListView.tpl.php

```diff
@@ -72,12 +72,14 @@
     </title>
     <meta name="description" content="Developed By sjpadgett@gmail.com">
     <?php
+    $csrf_php = js_escape(CsrfUtils::collectCsrfToken('doc-lib'));
+    $urlAjax = $GLOBALS['web_root'] . '/library/ajax/upload.php?parent_id=28&patient_id=' . attr_url($pid);
```

Comment:
odd to see parent_id hardcoded. What is this pointing to? If it is a specific category, then needs to go to a specific Category name (since upgraders will not always have same category/categoryId mapping. If need to do this (go to a specific category name rather than id), then would address in the upload.php script.

Reply:
I forgot I hardcoded. Thanks for the catch!